app/src/main/java/com/example/keybindhelper/ui/Projects/ProjectsFragment.kt

```diff
@@ -82,27 +82,27 @@ class ProjectsFragment : Fragment() {
         }
         )
 
-        return root;
+        return root
 
     }
     fun openKeybindFragment(){
         NavHostFragment.findNavController(this).navigate(R.id.nav_keybind)
     }
-    fun RefreshProjectList():List<Project>{
-        val rv= root.findViewById<RecyclerView>(R.id.projects_recycler);
+    fun refreshProjectList():List<Project>{
+        val rv= root.findViewById<RecyclerView>(R.id.projects_recycler)
         val projects=DatabaseManager.getOrderedProjects()
-        val searchBox=root.findViewById<EditText>(R.id.projects_search_edit);
-        if(searchBox.text.length>0) {
-            val filteredP= mutableListOf<Project>();
-            val search=searchBox.text.toString().lowercase();
+        val searchBox=root.findViewById<EditText>(R.id.projects_search_edit)
+        if(searchBox.text.isNotEmpty()) {
```

Comment:
Changed "length>0" to "isNotEmpty"
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/is-not-empty.html

Reply:
This was covered in a previously submitted code review.   When that occurs, first in wins.